### PR TITLE
[4] Rename recent BlankState feature to EmptyState

### PR DESCRIPTION
**Pull request by PhilETaylor**
The concept we are implementing is called Empty State design, therefore it makes sense to use that term throughout and not to confuse people with "Blank state/Blank slate" terms. Sorry, that was my fault. 

Therefore at this early stage it makes sense to get our own house in order from the start and use the term "Empty State" to refer to the "Empty State Design UX" we are employing. 

References (just a quick google):
 - https://www.toptal.com/designers/ux/empty-state-ux-design
 - https://material.io/design/communication/empty-states.html#content
 - https://www.justinmind.com/blog/everything-you-need-to-know-about-empty-state-design/
 - https://www.pinterest.com/weitseng/uiempty-state/

### CODE REVIEW

**Review by Bakual — APPROVED** [review]

**Comment by richard67** [comment]
That's the rename of the layouts. But what about these `$this->get('IsBlankSlate')` calls in the if conditions? Will the `IsBlankSlate` be renamed, too?

**Comment by PhilETaylor** [comment]
yeah I probably should do that too. Thanks.

**Comment by Quy** [comment]
Please rename the language constants too.

**Comment by PhilETaylor** [comment]
> Please rename the language constants too.

Done.

**Comment by PhilETaylor** [comment]
> Will the IsBlankSlate be renamed, too?

Done.

File: administrator/language/en-GB/com_banners.ini [hunk 1]
```diff
@@ -34,9 +34,9 @@ COM_BANNERS_BATCH_OPTIONS="Batch process the selected banners"
 COM_BANNERS_BATCH_TIP="If a category is selected for move/copy, any actions selected will be applied to the copied or moved banners. Otherwise, all actions are applied to the selected banners."
 COM_BANNERS_BEGIN_HINT="Begin date (yyyy-mm-dd)"
 COM_BANNERS_BEGIN_LABEL="Begin Date"
-COM_BANNERS_BLANKSTATE_BUTTON_ADD="Add your first banner"
-COM_BANNERS_BLANKSTATE_CONTENT="Banners are blocks of content containing advertising information and a record of the number of times the banner has been displayed is kept. You can use this component to create and manage your banners."
-COM_BANNERS_BLANKSTATE_TITLE="No Banners have been created yet."
+COM_BANNERS_EMPTYSTATE_BUTTON_ADD="Add your first banner"
```
Comment (Quy) [inline administrator/language/en-GB/com_banners.ini]: Sorry please alpha sort 😄

Comment (PhilETaylor) [reply administrator/language/en-GB/com_banners.ini]: hahahahah

Comment (PhilETaylor) [reply administrator/language/en-GB/com_banners.ini]: Done... maybe.

**Comment by PhilETaylor** [comment]
Conflicting files
administrator/components/com_banners/tmpl/banners/blankstate.php
administrator/components/com_contact/tmpl/contacts/blankstate.php
administrator/components/com_content/tmpl/articles/blankstate.php
administrator/components/com_messages/tmpl/messages/blankstate.php
administrator/components/com_newsfeeds/tmpl/newsfeeds/blankstate.php
administrator/components/com_privacy/tmpl/consents/blankstate.php
administrator/components/com_privacy/tmpl/requests/blankstate.php
administrator/components/com_users/tmpl/notes/blankstate.php
layouts/joomla/content/blankstate.php


grrrr ill take a look tonight

**Comment by PhilETaylor** [comment]
Rebased on `4.0-dev` to fix conflicts and force pushed.

**Comment by alikon** [comment]
feeling guilty... :fearful: 
but this task cannot be automated ?  (unit/system) test

**Comment by alikon** [comment]
nevermind
i've liked the idea,
didn't suspected to open a small pandora box...
me too lazy

File: layouts/joomla/content/emptystate.php [hunk 2]
```diff
@@ -33,12 +33,12 @@
 		<h1 class="display-5 fw-bold"><?php echo Text::_($textPrefix . '_BLANKSTATE_TITLE'); ?></h1>
```
Comment (Quy) [inline layouts/joomla/content/emptystate.php]:
```suggestion
		<h1 class="display-5 fw-bold"><?php echo Text::_($textPrefix . '_EMPTYSTATE_TITLE'); ?></h1>
```

Comment (PhilETaylor) [reply layouts/joomla/content/emptystate.php]: Thanks

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 6ed02ba1cb60eba7c9060f7313b6d3b6d8c1da21<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33334">issues.joomla.org/tracker/joomla-cms/33334</a>.</sub>

**Comment by Bakual** [comment]
> it was the great work by @Bakual that messed them up

You're welcome 🤣

**Comment by PhilETaylor** [comment]
Seriously thought - your help was much appreciated and this feature is all the more better for your contributions!

**Comment by Bakual** [comment]
It was a pleasure (really!) :)

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on 6ed02ba1cb60eba7c9060f7313b6d3b6d8c1da21<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33334">issues.joomla.org/tracker/joomla-cms/33334</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33334">issues.joomla.org/tracker/joomla-cms/33334</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**